src/main/java/org/reactome/release/qa/check/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities.java

```diff
@@ -19,18 +16,19 @@
 @SliceQATest
 public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities extends AbstractQACheck {
 
-    private static final String innateImmunityPathwayDbId = "168249";
+    private static final String INNATE_IMMUNITY_PATHWAY_DBID = "168249";
+    private static List<String> skiplistDbIds = new ArrayList<>();
```

Comment:
In this case, `static` should definitely be removed since `skiplistDbIds` will be receiving values in `executeQAChceck`.  Every instance of this check should have its own skip list and rather than shared using `static`.